earth_enterprise/src/third_party/gdal/README.txt

```diff
@@ -35,3 +35,11 @@ on a SLES 9 box (compatible with SLES 9,10, RHEL 4,5 and Ubuntu 6,8)
 
 To Build, you will need to build on a SLES 9 box using the scripts found in
 //depot/googleclient/geo/fusion/RPMS/build_scripts
+
+To build GDAL with MrSID Raster Format, MrSID library needs to be installed on your platform:
+1- Get MrSID Decoding SDK library
+2- Copy MrSDK library into a folder. Ex: /opt
+3- Ensure that 'libltidsdk.so' can be disovered by 'ldconfig' command:
+	3.1- Add the full path of 'libltidsdk.so' to ‘/etc/ld.so.conf’. You can edit that file as root admin only.
+	3.2- Rebuild the libraries path cache (‘/etc/ld.so.cache’): $sudo ldconfig
```

Comment:
These commands should be surrounded by backticks "`" so that they will be properly formatted for markdown and helps them to stand out.

Reply:
Oh, I see, this isn't actually a markdown file, so the formatting doesn't work. I guess it doesn't matter either way then.